src/app/components/mno-end-wizard/mno-end-wizard.less

```diff
@@ -0,0 +1,34 @@
+#wizard-end{
+  min-height: 85%;
+  max-height: 85%;
```

Comment:
@alexnoox 
I have read that using max-height and min-height, I gain more control. In fact, when I switch this to height:85%, the overflow hidden scrollable stop working and therefore the elements come out the box.